### DISCUSSION TITLE
Why does `n-input` have no border?

### DISCUSSION THREAD

**Discussion by mitya33** [tusen-ai/naive-ui#4974] [General]
I've spent an hour on this and cannot figure out why my `n-input` element has no border.

I've reduced my app to just the following:

**App.vue**

```
<template>
	<Foo />
</template>

<script setup>
import Foo from './components/Foo.vue'
</script>
```

**Foo.vue**

```
<template>
  <n-input v-model:value="value" type="text" placeholder='Basic input' />
```

...and I still get borderless.

![image](https://github.com/tusen-ai/naive-ui/assets/30174698/3b40931d-1dee-4680-b624-ee19ea5f003a)

I've tried to force a border manually:

**App.vue**

```
<template>
	<n-config-provider :theme-overrides='theme'>
		<Foo />
	</n-config-provider>
</template>

<script setup>
import Foo from './components/Foo.vue'
const theme = {
	Input: {
		border: 'solid 2px red', //<-- does nothing
		placeholderColor: 'blue' //<-- works
	}
}
</script>
```

What am I doing wrong? I can't reproduce this in sandbox, so it must be something about my install.

Thank you in advance.

**Comment by Sepush** [comment]
did you use tailwind or unocss if so follow the docs guide https://www.naiveui.com/en-US/os-theme/docs/style-conflict
